stockflux-chart/src/App.js

```diff
@@ -12,79 +12,74 @@ const ALL = { uuid: '*' };
+  }
 
-    const onNewsClick = () => {
-        if (symbol) {
-            Intents.viewNews(symbol, name);
-        }
+  const { data } = useInterApplicationBusSubscribe(
+    parentUuid ? parentUuid : ALL,
+    'stockFluxChart:' + listenerSymbol
+  );
+  if (data && data.message) {
+    if (data.message.symbol && symbol !== data.message.symbol) {
+      setSymbol(data.message.symbol);
     }
-
-    const onWatchlistClick = () => {
-        if (symbol && name) {
-            Intents.addWatchlist(symbol, name);
-        }
+    if (data.message.name && name !== data.message.name) {
+      setName(data.message.name);
     }
+  }
 
-    const getData = (symbol) => {
-        if (symbol) {
-            chart.changeStockFluxProduct(symbol);
-        }
+  const getData = symbol => {
+    if (symbol) {
+      chart.changeStockFluxProduct(symbol);
     }
+  };
 
-    return (
-        <>
-            <div className='main'>
-                <div className='main-content'>
-                    <Components.Titlebar />
-                    <div id="showcase-title">
-                        {symbol && (
-                            <div className="code">
-                                {symbol}
-                            </div>
-                        )}
-                        <div className="name">
-                            {name ? name : 'Generated Data'}
-                        </div>
-                        <div className="chart-nav-icons">
-                            <div className={"chart-nav-icon" + (symbol ? '' : ' icon-disabled')} onClick={onNewsClick}>
-                                <Components.News />
-                            </div>
-                            <div className={"chart-nav-icon" + (symbol ? '' : ' icon-disabled')} onClick={onWatchlistClick}>
-                                <Components.Watchlist />
-                            </div>
-                            <div className={"chart-nav-icon" + (symbol ? '' : ' icon-disabled')} onClick={() => getData(symbol)}>
-                                <FaSyncAlt />
-                            </div>
-                        </div>
-                    </div>
-                    <Chart getData={getData} chart={chart} symbol={symbol}/>
-                </div>
+  return (
+    <>
+      <div className="main">
+        <div className="main-content">
+          <Components.Titlebar />
+          <div id="showcase-title">
+            {symbol && <div className="code">{symbol}</div>}
+            <div className="name">{name ? name : 'Generated Data'}</div>
+            <div className="chart-nav-icons">
+              <div
+                className={'chart-nav-icon' + (symbol ? '' : ' icon-disabled')}
+              >
+                <Components.NewsShortcut symbol name />
+              </div>
+              <div
+                className={'chart-nav-icon' + (symbol ? '' : ' icon-disabled')}
+              >
+                <Components.WatchlistShortcut />
```

Comment:
Did you mean to add `symbol` and `name` here?

Reply:
Yes, indeed! Thanks for pointing this out. Fixed -> 601e93446b2f3c27af09e3263b6b03090386bcab